docs/roadmap-2021.md

```diff
@@ -1,14 +1,42 @@
+### Roadmaps
+- [Eventing Working Group](#Eventing)
+- [Eventing Sources Working Group](#Eventing-Sources)
+- [Eventing Kafka Working Group](#Eventing-Kafka)
+
 # Eventing
 
 ## Next 3 Months (Short-term)
 
+### Spec Solidification
+
+Review and update documented specification to reflect the existing
+implementation (v1 API).
```

Comment:
Naive assumption 😁 but this implies uni-directional change Implementation -> Specs rather than bi-directional. Is that incorrect ?